src/mon/Monitor.cc

```diff
@@ -551,29 +551,29 @@ int Monitor::preinit()
-    pcb.add_u64(l_cluster_num_osd, "num_osd");
-    pcb.add_u64(l_cluster_num_osd_up, "num_osd_up");
-    pcb.add_u64(l_cluster_num_osd_in, "num_osd_in");
-    pcb.add_u64(l_cluster_osd_epoch, "osd_epoch");
-    pcb.add_u64(l_cluster_osd_bytes, "osd_bytes");
-    pcb.add_u64(l_cluster_osd_bytes_used, "osd_bytes_used");
-    pcb.add_u64(l_cluster_osd_bytes_avail, "osd_bytes_avail");
-    pcb.add_u64(l_cluster_num_pool, "num_pool");
-    pcb.add_u64(l_cluster_num_pg, "num_pg");
-    pcb.add_u64(l_cluster_num_pg_active_clean, "num_pg_active_clean");
-    pcb.add_u64(l_cluster_num_pg_active, "num_pg_active");
-    pcb.add_u64(l_cluster_num_pg_peering, "num_pg_peering");
-    pcb.add_u64(l_cluster_num_object, "num_object");
-    pcb.add_u64(l_cluster_num_object_degraded, "num_object_degraded");
-    pcb.add_u64(l_cluster_num_object_misplaced, "num_object_misplaced");
-    pcb.add_u64(l_cluster_num_object_unfound, "num_object_unfound");
-    pcb.add_u64(l_cluster_num_bytes, "num_bytes");
-    pcb.add_u64(l_cluster_num_mds_up, "num_mds_up");
-    pcb.add_u64(l_cluster_num_mds_in, "num_mds_in");
-    pcb.add_u64(l_cluster_num_mds_failed, "num_mds_failed");
-    pcb.add_u64(l_cluster_mds_epoch, "mds_epoch");
+    pcb.add_u64(l_cluster_num_mon, "num_mon", "Number of monitors, registered on cluster");
+    pcb.add_u64(l_cluster_num_mon_quorum, "num_mon_quorum", "Number of monitors in quorum");
+    pcb.add_u64(l_cluster_num_osd, "num_osd", "Total number of OSD");
+    pcb.add_u64(l_cluster_num_osd_up, "num_osd_up", "Number of working OSD");
+    pcb.add_u64(l_cluster_num_osd_in, "num_osd_in", "Number of OSD in state \"in\" (they are in cluster)");
+    pcb.add_u64(l_cluster_osd_epoch, "osd_epoch", "Current epoch of OSD map");
+    pcb.add_u64(l_cluster_osd_bytes, "osd_bytes", "Total number of bytes on cluster");
+    pcb.add_u64(l_cluster_osd_bytes_used, "osd_bytes_used", "Number of used bytes on cluster");
+    pcb.add_u64(l_cluster_osd_bytes_avail, "osd_bytes_avail", "Number of available bytes on cluster");
+    pcb.add_u64(l_cluster_num_pool, "num_pool", "Number of pools");
+    pcb.add_u64(l_cluster_num_pg, "num_pg", "Total number of placement groups");
+    pcb.add_u64(l_cluster_num_pg_active_clean, "num_pg_active_clean", "Number of placement groups in active+clean (usual) state");
+    pcb.add_u64(l_cluster_num_pg_active, "num_pg_active", "Number of placement groups in active state");
+    pcb.add_u64(l_cluster_num_pg_peering, "num_pg_peering", "Number of placement groups in peering process");
+    pcb.add_u64(l_cluster_num_object, "num_object", "Total number of objects on cluster");
+    pcb.add_u64(l_cluster_num_object_degraded, "num_object_degraded", "Number of degraded (missing replicas) objects");
+    pcb.add_u64(l_cluster_num_object_misplaced, "num_object_misplaced", "Number of misplaced (wrong location in the cluster) objects");
+    pcb.add_u64(l_cluster_num_object_unfound, "num_object_unfound", "Number of currently unfound objects");
```

Comment:
I think removing **currently** would be best. Otherwise it suggests that another counter exists to  show past unfound object or something similar.